src/app/wallet/settings/settings.component.html

```diff
@@ -444,7 +444,7 @@ <h4 class="option">Map port using UPnP</h4>
 
           <div class="item">
             <h4 class="option">Port forwarding</h4>
-            <p class="desc">Alternatively, you can configure your router to forward port <code>51738</code></p>
+            <p class="desc">Alternatively, you can configure your router to forward port<!-- <code>51738</code> --></p>
```

Comment:
Why did you hide this? How are users supposed to port forward, when they don't know the port?

Reply:
The port does change, depending on whether you are on testnet or mainnet :)
Also, while its not likely to be used outside of mainnet/testnet, the daemon can be configured with the 
`-port=<>`
flag, which does change the port value. This is also possible to be configured via the Desktop by passing it in as a flag when starting the Desktop client (or the daemon might have been configured via a particl.conf file containing the value).